spec/forms/serials_form_spec.rb

```diff
@@ -30,13 +30,13 @@
                 { value: 'My Serial', type: 'main title' },
                 { value: '7', type: 'part number' },
                 { value: 'samurai', type: 'part name' }
-              ],
-              note: [
-                { value: '1990', type: 'date/sequential designation' }
               ]
             }
           ],
-          'purl' => "https://purl.stanford.edu/#{druid.delete_prefix('druid:')}"
+          note: [
```

Comment:
is it intended that you changed the level at which the note is present in the cocina object?

Reply:
Yes, that's the change.

Reply:
<img width="961" alt="Screen Shot 2022-03-31 at 2 24 34 PM" src="https://user-images.githubusercontent.com/92044/161133993-3656fb51-29b6-48e9-b8db-f833aa09007b.png">